packages/collaboration-extension/src/index.ts

```diff
@@ -0,0 +1,204 @@
+ */
+const menuBarPlugin: JupyterFrontEndPlugin<void> = {
+  id: '@jupyterlab/collaboration-extension:userMenuBar',
+  autoStart: true,
+  requires: [ICurrentUser, IUserMenu],
+  activate: (
+    app: JupyterFrontEnd,
+    user: ICurrentUser,
+    menu: IUserMenu
+  ): void => {
+    const { shell } = app;
+
+    if (PageConfig.getOption('collaborative') !== 'true') {
+      return;
+    }
+
+    const menuBar = new MenuBar({
+      forceItemsPosition: {
+        forceX: false,
+        forceY: false
+      },
+      renderer: new RendererUserMenu(user)
+    });
+    menuBar.id = 'jp-UserMenu';
+    user.changed.connect(() => menuBar.update());
+    menuBar.addMenu(menu as Menu);
+    shell.add(menuBar, 'top', { rank: 1000 });
+  }
+};
+
+/**
+ * Jupyter plugin creating a global awareness for RTC.
+ */
+const rtcGlobalAwarenessPlugin: JupyterFrontEndPlugin<IAwareness> = {
+  id: '@jupyterlab/collaboration-extension:rtcGlobalAwareness',
+  autoStart: true,
+  requires: [ICurrentUser, IStateDB],
+  provides: IGlobalAwareness,
+  activate: (
+    app: JupyterFrontEnd,
+    currentUser: User,
+    state: StateDB
+  ): IAwareness => {
+    const ydoc = new Y.Doc();
+
+    if (PageConfig.getOption('collaborative') !== 'true') {
+      return new AwarenessMock(ydoc);
+    }
+
+    const awareness = new Awareness(ydoc);
+
+    const server = ServerConnection.makeSettings();
+    const url = URLExt.join(server.wsUrl, 'api/yjs');
+
+    new WebsocketProvider(url, 'JupyterLab:globalAwareness', ydoc, {
+      awareness: awareness
+    });
+
+    const userChanged = () => {
+      const name =
+        currentUser.displayName !== ''
+          ? currentUser.displayName
+          : currentUser.name;
+      awareness.setLocalStateField('user', { ...currentUser.toJSON(), name });
+    };
+    if (currentUser.isReady) {
+      userChanged();
+    }
+    currentUser.ready.connect(userChanged);
+    currentUser.changed.connect(userChanged);
+
+    state.changed.connect(async () => {
+      const data: any = await state.toJSON();
+      const current = data['layout-restorer:data']?.main?.current || '';
+
+      if (current.startsWith('editor') || current.startsWith('notebook')) {
+        awareness.setLocalStateField('current', current);
+      } else {
+        awareness.setLocalStateField('current', null);
+      }
+    });
+
+    return awareness;
+  }
+};
+
+/**
+ * Jupyter plugin adding the RTC information to the application left panel if collaborative flag enabled.
+ */
+const rtcPanelPlugin: JupyterFrontEndPlugin<void> = {
+  id: '@jupyterlab/collaboration-extension:rtcPanel',
+  autoStart: true,
+  requires: [ICurrentUser, IGlobalAwareness],
+  provides: IUserPanel,
+  activate: (
+    app: JupyterFrontEnd,
+    currentUser: User,
+    awareness: Awareness
+  ): void => {
+    if (PageConfig.getOption('collaborative') !== 'true') {
+      return;
+    }
+
+    const userPanel = new AccordionPanel({
+      renderer: new RTCPanel.Renderer()
+    });
```

Comment:
There is now a `SidePanel` that contains a `AccordionPanel` with the proper renderer in `ui-components`:

```suggestion
    const userPanel = new SidePanel();
```